### PR TITLE
add more specific description for doc

### PR DESCRIPTION
**Pull request by laithiennhan**
update README.md

### CODE REVIEW

File: README.md
Comment (matchalatte2609) [inline README.md:1]: need to fix

File: README.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # inventory-management
-This is a prototype
+This is a prototype for an inventory management application
```
Comment (laithiennhan) [inline README.md:2]: looking sus

Comment (laithiennhan) [reply README.md:2]: solved

**Review by matchalatte2609 — APPROVED** [review]
lgtm

**Pull request merged**